pywbem/_logging.py

```diff
@@ -14,404 +14,341 @@
+  parameter defines which of the pywbem loggers is being configured. This
+  configure step does not activate any pywbem logger. If no loggers are
+  configured, pywbem defaults to a logger with the Python
+  logging NullHandler.
+
+* **Set the log record detail level for pywbem loggers(Optional)** - Because
+  pywbem logs can generate large quantities of information, they allow the user
+  to control the quantity of information in each log record.  This is a pywbem
+  feature so it can only be configured using the pywbem methods defined in
+  :ref:`Logging configuration methods` using the `detail_level` parameter. The
+  `detail_level` possible values are defined in
+  :data:`~pywbem_logging.LOG_DETAIL_LEVELS`. The default for `detail_level` is
+  `all` which logs all if the data in the requests and responses if the
+  `detail_level` parameter is not set.
+
+* **Activate the pywbem loggers** - A pywbem logger is activated when it is
+  connected to one or more WBEMConnections so that log records will be generated
+  when any of the WBEMConnection methods that communicates with a WBEM server
+  is executed for that WBEMConnection. Pywbem separates the configuration
+  of the loggers from activation to allow logger configuration by either pywbem
+  methods or standard Python logging methods. Activation is controlled only by
+  the `connection` parameter of  the methods defined in
+  :ref:`Logging configuration methods` which either activates logging for
+  all (`connect=True`) subsequently created :class:`~pywbem.WBEMConnection`
+  objects or for a particular connection (`connect=conn` where conn is a
+  WBEMConnection object).
+
+.. _`Logging configuration methods`:
+
+Logging configuration methods
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+The pywbem loggers may be configured through the following pywbem methods and
+must be used to set the log detail_level and activate the loggers.
+
+  * :meth:`~pywbem.WBEMConnection.configure_logger` - Configures and
+    optionally activates one or more of the loggers that log user-issued calls
+    to :class:~pywbem.WBEMConnection methods that drive WBEM operations and
+    pywbem http requests and responses. Thus, for example, the following
+    would configure the api logger as a file logger for a single connection::
+
+       conn = WBEMConnection(...)
+       WBEMConnection.configure_logger('api', dest=file, log_filename='blah.log'
+                                       connection=conn)
+
+  * :func:`configure_loggers_from_string` - Allows configuring and
+    activating the pywbem loggers from a single formatted string. This is most
+    useful in defining the loggers from a command line tool such as pywbemcli.
+    Thus, for example, the following could be used to configure and
+    activate both loggers::
+
+        configure_loggers_from_string('api=file:summary,http=stderr:all',
+                                      log_filename='mylog.log',
+                                      connection='True')
```

Comment:
We have an immediately following example section. Not sure we need the example at this place in addition.

Reply:
DONE. Removed

Reply:
Note that I removed the example from both method definitions